utils/pckgen

```diff
@@ -13,19 +13,27 @@
 
 
 CAS_NAME="open-cas-linux"
+CAS_HOMEPAGE="https://open-cas.github.io"
+CAS_GIT="https://github.com/Open-CAS/open-cas-linux.git"
+CAS_LICENSE_NAME="BSD-3-Clause-Clear"
 SUPPORTED_FROM_VERSION="20.03"
 THIS=$(basename "$0")
 ARCH="$(uname -i)"
 SYS_INFO="/etc/os-release"
 SCRIPT_BASE_DIR=$(dirname $(realpath "$0"))
-BASE_SPEC="$SCRIPT_BASE_DIR/$CAS_NAME.spec.base"
+RPM_SPEC_FILE="$SCRIPT_BASE_DIR/$THIS.d/rpm/CAS_NAME.spec"
+DEB_CONTROL_FILES_DIR="$SCRIPT_BASE_DIR/$THIS.d/deb/debian"
+PACKAGE_MAINTAINER="Rafal Stefanowski <rafal.stefanowski@intel.com>"
+PACKAGE_DATE="$(date -R)"
 TEMP_TEMPLATE="opencas-${THIS}"
-DEPENDENCIES=(sed git mktemp rsync)
+DEPENDENCIES=(git mktemp rsync sed)
 # Dependencies for particular packages creation:
 DEPENDENCIES_TAR=(tar)
 DEPENDENCIES_ZIP=(zip)
-DEPENDENCIES_RPM=(tar rpmbuild)
+DEPENDENCIES_RPM=(rpmbuild tar)
 DEPENDENCIES_SRPM=("${DEPENDENCIES_RPM[@]}")
+DEPENDENCIES_DEB=(debuild dh fakeroot tar)
```

Comment:
When `debuild` and `dh` are missing, the following message is printed:
```
--- Checking for dependencies
ERROR: pckgen: Dependencies not installed. You need to provide these programs first: debuild dh
```
which is only partially helpful, because `apt install debuild dh` fails:
```
E: Unable to locate package debuild
E: Unable to locate package dh
```
as those programs are provided by packages named `devscripts` and `debhelper` respectively.